backend/src/modules/email-sending/core/application/command/SendEmailCommandHandler.ts

```diff
@@ -0,0 +1,34 @@
+import { CommandHandler } from '../../../../../shared/core/application/command/CommandHandler';
+import { SendEmail } from './SendEmail';
+import { CommandResult } from '../../../../../shared/core/application/command/CommandResult';
+import NodeMailer from 'nodemailer';
+
+export class SendEmailCommandHandler implements CommandHandler<SendEmail> {
+  constructor() {}
+
+  async execute(command: SendEmail): Promise<CommandResult> {
+    // const transporter = NodeMailer.createTransport({
+    //     host: "localhost",
+    //     port: 1025
+    // });
+
+    const transporter = NodeMailer.createTransport({
+      host: 'smtp.gmail.com',
+      port: 465,
+      secure: true,
+      auth: {
```

Comment:
Tak, a takich rzeczy nie commitujemy, szczegolnie w publicznym repo :) Dlatego defaultowo dobrze jest dostarczyć np. implementację wysyłania maili tylko logującego mail na konsoli, żeby było można odpalić apkę nie potrzebując serwera SMTP (tak samo jak nie potrzebujemy bazy danych, to mamy różne implementacje Repository). 

A potem takie rzeczy trzyma się w plikach .env dla konkretnego środowiska i przekazuje bardziej "poufnie" :) /